.github/workflows/validate.yml

```diff
@@ -22,6 +23,7 @@ jobs:
         with:
           # must be specified without patch version
           version: v1.42
+          args: --enable gosec
```

Comment:
I _guess_ this is done by adding the `gosec` to  `linters:` / `enable:` section of `.golang.yml` file.

Reply:
Note: If configuring through `args:` becomes too complicated, we could consider adding a `.golangci-lint.yaml` (I personally find that more convenient, as it also allows documenting the options we use through comments)

Reply:
oh, lol, our comments crossed (and forgot we already have the config file)

Reply:
Yep, found the yaml file already. Somehow the action finds more than the command but as [it does not tell where](https://github.com/golangci/golangci-lint-action/issues/119) ...

Reply:
Hm, it finds issues with confidence: HIGH and severity: MEDIUM, though I filter the other way around. I'll set both to medium and fix or nosec the findings.

Reply:
Yes, it doesn't show the line numbers in the logs (which is definitely a pity), but it _does_ annotate the code in the PR;

<img width="877" alt="Screenshot 2021-11-12 at 10 34 37" src="https://user-images.githubusercontent.com/1804568/141444860-b4c34454-d860-4ba4-97b5-28caa8b248ee.png">

Reply:
Nice, that helps.

Reply:
(I would definitely love to see that linked issue fixes as well though, as I think they should not be mutually exclusive 😅, but that's an issue we already have with the linter, so not something new)